homeassistant/components/google_assistant/helpers.py

```diff
@@ -1,5 +1,5 @@
 """Helper classes for Google Assistant integration."""
-from asyncio import gather
+import asyncio
```

Comment:
Please remove the Google Assistant changes from this PR and focus on envisalink only. Google changes can be doen in a future PR.